recipes-openxt/openxt-measuredlaunch/openxt-measuredlaunch_1.0.bb

```diff
@@ -35,4 +35,5 @@ RDEPENDS_${PN} = " \
     tboot-utils \
     tboot-pcr-calc \
     openxt-keymanagement \
+    xenclient-tpm-scripts \
```

Comment:
Make openxt-keymanagement RDEPEND on xenclient-tpm-scripts instead of adding it explicitly.

Reply:
`seal-system` does not use `key-functions`.
So it seems preferable to leave `xenclient-tpm-scripts` in here (`tpm-functions` is sourced in `seal-system`) and add `xenclient-tpm-scripts` as `RDEPENDS` to `openxt-keymanagement` as well.